src/nxt_http_route_addr.c

```diff
@@ -47,6 +47,16 @@ nxt_http_route_addr_pattern_parse(nxt_mp_t *mp,
 
     nxt_str_null(&port);
```

Comment:
Nulling port variable has no sense in case of "unix" socket. This should be after the "unix" socket condition, not before.

Reply:
Oops.  Right.

Reply:
It makes sense to move "unix" socket condition also above the `addr.length < 2` check. Because that check is directly related to the following condition with `addr.start[0] == '*' && addr.start[1] == ':'`. Keeping them together makes code self-explainable about why length should be not less than 2.